src/JoeScan.Pinchot.Shared/ProfileAssembler.cs

```diff
@@ -89,7 +89,6 @@ internal void AssembleProfiles(ProfileFragments fragments)
             p.RawPointsMemory = new Memory<Point2D>(rawPointsArray, rawPointsArrayIndex * Globals.RawProfileDataLength,
                 Globals.RawProfileDataLength);
             var rawPointsSpan = p.RawPointsMemory.Span;
-            rawPointsSpan.Fill(new Point2D(double.NaN, double.NaN, Globals.ProfileDataInvalidBrightness));
```

Comment:
I like this. I'll review to make sure changing the defaults for `Point2D` don't mess with anything else.

Reply:
It does.  See the property initializers on the Point2D